mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -31,13 +52,42 @@ AUI.add(
 									var listItem = currentTarget.ancestor('li');
 
 									listItem.remove();
+
+									if (finishedTask) {
+
+										var testHtml = A.Lang.sub(
```

Comment:
Names of variables should hold a meaning, 'testHtml' is a placeholder name I assume.